constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ConstraintLayoutBaseScope.kt

```diff
@@ -678,6 +678,463 @@ abstract class ConstraintLayoutBaseScope internal constructor(extendFrom: CLObje
         return ref
     }
 
+    /**
+     * Grid helpers make build a Row representation more intuitively.
+     *
+     * @param elements [LayoutReference]s to be laid out by the Flow helper
+     * @param elements [LayoutReference]s to be laid out by the Flow helper
+     * @param orientation 0 if horizontal and 1 if vertical
+     * @param verticalGap defines the gap between views in the y axis
+     * @param horizontalGap defines the gap between views in the x axis
+     * @param rowWeights defines the weight of each row
+     * @param columnWeights defines the weight of each column
+     * @param skips defines the positions in a Grid to be skipped
+     * @param spans defines the spanned area(s) in Grid
+     * @param padding sets padding around the content
+     */
+    fun createRow(
+        vararg elements: LayoutReference?,
```

Comment:
No need for this to be Nullable.: `vararg elements: LayoutReference`

Reply:
Updated.